pylops/avo/avo.py

```diff
@@ -290,7 +325,11 @@ def approx_zoeppritz_pp(vp1, vs1, rho1, vp0, vs0, rho0, theta1):
     return rpp
 
 
-def akirichards(theta, vsvp, n=1):
+def akirichards(
+    theta: npt.ArrayLike,
+    vsvp: Union[float, npt.ArrayLike],
+    n: int = 1,
+) -> Union[npt.ArrayLike, npt.ArrayLike, npt.ArrayLike]:
```

Comment:
Should be Tuple[npt.Arraylike, ...]

Reply:
I kept making this mistake, I checked now for `-> U` and fixed all of them :)